docs/source/json-api/index.rst

```diff
@@ -72,9 +72,9 @@ From a DAML project directory:
       --address <value>
             IP address that HTTP JSON API service listens on. Defaults to 127.0.0.1.
       --http-port <value>
-            HTTP JSON API service port number. A port number of 0 will let the system pick up an ephemeral port. Consider specifying port-file option with port number 0.
+            HTTP JSON API service port number. A port number of 0 will let the system pick an ephemeral port. Consider specifying `--port-file` option with port number 0.
```

Comment:
Isn't it like this in RST?
```suggestion
            HTTP JSON API service port number. A port number of 0 will let the system pick an ephemeral port. Consider specifying ``--port-file`` option with port number 0.
```

Reply:
yes, it is like this in rst, but this is a copy/paste from `daml json-api --help`.